library/src/morpheus-master/sass/modules/tool/conversations/_sakai-topic-summary.scss

```
@@ -139,18 +136,13 @@
   .new-posts-number {
     font-size: 10px;
     font-weight: 500;
-    color: #176EA3;
+    color: var(--sakai-text-color);
   }
 
   .selected {
-    background: #D4EBF9;
-    border: 1px solid #D4EBF9;
-  }
-  .selected * {
-    color: #666666;
-  }
+    background: var(--sakai-background-color-4);
+    border: 1px solid var(--sakai-border-color);
 
-  .selected {
     .discussion-icon {
       color: #FFFFFF;
```

I'd suggest using `--sakai-text-color-inverse` in case someone wanted to use a crazy background color that made white an issue here.
```suggestion
      color: var(--sakai-text-color-inverse);
```